blogposts/2021/career-framework.md

```diff
@@ -218,9 +218,7 @@ Our framework values efficiency. Any qualitative feedback for an IC “should be
 
 [IC4](https://handbook.sourcegraph.com/engineering/career-development/framework#ic4) describes “A particularly experienced, impactful contributor.”
 
-[IC5](https://handbook.sourcegraph.com/engineering/career-development/framework#ic5) describes “A different IC role from levels 1-4, with a focus on technical leadership.”
-
-[IC6](https://handbook.sourcegraph.com/engineering/career-development/framework#ic6) and [IC7](https://handbook.sourcegraph.com/engineering/career-development/framework#ic7) are deliberately left without bullet points, having not yet been finalized.
+[IC5](https://handbook.sourcegraph.com/engineering/career-development/framework#ic5) and [IC6](https://handbook.sourcegraph.com/engineering/career-development/framework#ic6) are deliberately left without bullet points because the impact an IC5 and IC6 makes will vary between individuals. For IC5, we've put together a [list of profiles](https://handbook.sourcegraph.com/departments/product-engineering/engineering/career-development/ic5/) that evoke common traits amongst IC5 engineers. Here, we encourage engineers to use these profiles in "career growth conversations between ICs and EMs, and in calibration discussions between EMs and directors."
```

Comment:
```suggestion
[IC5](https://handbook.sourcegraph.com/engineering/career-development/framework#ic5) and [IC6](https://handbook.sourcegraph.com/engineering/career-development/framework#ic6) are deliberately left without bullet points because the impact an IC5 or IC6 makes will vary between individuals. For IC5, we've put together a [list of profiles](https://handbook.sourcegraph.com/departments/product-engineering/engineering/career-development/ic5/) that evoke common traits amongst IC5 engineers. Here, we encourage engineers to use these profiles in "career growth conversations between ICs and EMs, and in calibration discussions between EMs and directors."
```